README.md

```diff
@@ -440,6 +457,7 @@ Please [discover modm's peripheral drivers for your specific device][discover].
 <td align="center">✗</td>
 <td align="center">✗</td>
 <td align="center">✗</td>
+<td align="center">✗</td>
```

Comment:
V70 **does** support this, but it's provided by Enhanced Embedded Flash Controller (see datasheet, section 11.1.5.6). Again, what should I do here?

Reply:
Should I leave it as-is like system clock then?

Reply:
Leave it for now, I'll check all of this naming stuff out this week.